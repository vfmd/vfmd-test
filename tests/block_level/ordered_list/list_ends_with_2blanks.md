 1. First list

 1. First list


 1. Second list

 1. Second list

    Part of the list

 1. Second list

 1. Second list


    Code block
