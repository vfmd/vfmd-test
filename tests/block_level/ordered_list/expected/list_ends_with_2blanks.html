<ol>
<li><p>First list</p></li>
<li><p>First list</p></li>
</ol>
<ol>
<li><p>Second list</p></li>
<li><p>Second list</p>

<p>Part of the list</p></li>
<li><p>Second list</p></li>
<li><p>Second list</p></li>
</ol>

<pre><code>Code block
</code></pre>
